app/views/full-page-examples/applicant-details/index.njk

```diff
@@ -135,6 +135,7 @@ notes: >-
 {% endblock %}
 
 {% block body_end %}
+  {% include "partials/legacyJavaScript.njk" %}
```

Comment:
Do you think it's worth exploring having a layout for legacy examples?

Reply:
Yeah we could definitely refactor the views in the demo apps...

Reply:
Non-blocking, happy to leave it for now if you think it makes sense 👍

Reply:
Refactored it a bit, still more could do I imagine.